examples/sdk-usage/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java'
+}
+
+group 'io.opentelemetry'
+version '0.2.0'
+
+//create a single Jar with all dependencies
+task fatJar(type: Jar) {
+    baseName = project.name + '-all'
+    from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+    with jar
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'io.opentelemetry:opentelemetry-sdk:0.2.0'
+    implementation 'io.opentelemetry:opentelemetry-exporters-inmemory:0.2.0'
```

Comment:
we should depend on 0.3.0-SNAPSHOT until we get that published, since these examples look pretty out of date. thoughts?

Reply:
I'm fine either way but also tend towards using 0.3.0.